src/types/TodoItemType.ts

```diff
@@ -0,0 +1,5 @@
+export type TodoItemType = {
```

Comment:
```suggestion
export type TodoItem = {
```
in case you have names conflict in the file where you import this you may do the following:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html